src/plugins/vis_type_timelion/public/timelion_vis_renderer.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { lazy } from 'react';
+import { render, unmountComponentAtNode } from 'react-dom';
+
+import { KibanaContextProvider } from '../../kibana_react/public';
+import { ExpressionRenderDefinition } from '../../expressions/common/expression_renderers';
+import { VisualizationContainer } from '../../visualizations/public';
+import { TimelionVisDependencies } from './plugin';
+import { TimelionRenderValue } from './timelion_vis_fn';
+// @ts-ignore
+const TimelionVisComponent = lazy(() => import('./components/timelion_vis_component'));
+
+export const getTimelionVisRenderer: (
+  deps: TimelionVisDependencies
+) => ExpressionRenderDefinition<TimelionRenderValue> = (deps) => ({
+  name: 'timelion_vis',
+  displayName: 'Timelion visualization',
```

Comment:
should this value be localized?

Reply:
This is a good question.
Despite the fact that docs says:
```
  /**
   * A user friendly name of the renderer as will be displayed to user in UI.
   */
  displayName: string;
```
I can't find any usage of the `displayName` in `src/plugins`, so this never affects our basic plugins.
But this could be an option for `canvas` app.
I had a talk with @streamich around the `ExpressionRenderer` keys, and he'll define their necessity after their team meeting today.
For now, we decided to make it an optional property.